packages/core/src/codecatalyst/commands.ts

```diff
@@ -26,6 +26,7 @@ import { CreateDevEnvironmentRequest, UpdateDevEnvironmentRequest } from 'aws-sd
 import { Auth } from '../auth/auth'
 import { SsoConnection } from '../auth/connection'
 import { isInDevEnv, isRemoteWorkspace } from '../shared/vscode/env'
+import { commandPalette } from '../codewhisperer/commands/types'
```

Comment:
the /codewhisperer/ dir is a hint that this constant should be moved somewhere higher. `packages/core/src/shared/telemetry/telemetry.ts` seems like a good place.